capstone-ui/src/Components/Feed/Feed.jsx

```diff
@@ -38,13 +48,22 @@ export default function Feed ( { loggedin }) {
       }
     }, [loggedin, user.stances, user.following]);
 
+    useEffect(() => {
+      setCurrentPage(1);
+      fetchData(1);
```

Comment:
Feels a bit redundant to reset the state to value of 1. Is it possible to combine this useEffect with the one below that handles fetching on currentPage change?